mkdd_editor.py

```diff
@@ -1,5 +1,8 @@
 import traceback
 import os
+import sys
+cwd = sys.argv[0].rsplit('\\', 1)[0]  # changes current working directory to always be on the editor's directory as every paths are hardcoded to be loaded from here
```

Comment:
I always wanted to make this change, as I keep forgetting to change to the correct directory.

We might need to tweak your approach to make it cross-platform; in Unix systems use forward slashes.

Suggestion:

```python
script_path = os.path.realpath(__file__)
script_dir = os.path.dirname(script_path)
os.chdir(script_dir)
```

Also, I think these lines could be moved down below, at the start of the `if __name__ == "__main__":` block, to avoid executing code if the file is imported as a module.

Reply:
these lines can't be moved down below because it's importing local scripts at the beginning
first one you can see for example is the "import py_obj"